lib/unleash/feature_toggle.rb

```diff
@@ -72,13 +72,12 @@ def get_variant(context, fallback_variant = disabled_variant)
 
     # only check if it is enabled, do not do metrics
     def am_enabled?(context, default_result)
```

Comment:
I would like to simplify a bit the `am_enabled?` method, so it is easier to understand.

If you would like, you could re-write it to look a bit more like this?

```ruby
    # only check if it is enabled, do not do metrics
    def am_enabled?(context, default_result)
      result =
        if self.enabled
          self.strategies.empty? ||
          self.strategies.select{ |s|
            r = Unleash::STRATEGIES.fetch(s.name.to_sym, :unknown).is_enabled?(s.params, context)
            Unleash.logger.debug "Strategy #{s.name} returned #{r} with context: #{context}" #"for params #{s.params} "
            r
          }.any?
        else
          default_result
        end

      Unleash.logger.debug "FeatureToggle (enabled:#{self.enabled} default_result:#{default_result} and Strategies combined returned #{result})"
      return result
    end
```

Reply:
sure, will patch it later.

Reply:
fix on commit d43f0e6f985e9ba801afd3881ab747caee77e148